### DISCUSSION TITLE
Setting up a V3 without an account

### DISCUSSION THREAD

**Discussion by jedahan** [gtxaspec/wz_mini_hacks#690] [Q&A]
How can I setup a V3 without making a wyze account? It seems this is a necessary step before giving wifi credentials, which also seems necessary to install the wz_mini_hacks repo.

Alternatively, can I use a usb-ethernet to make a local network and ssh in direct from my laptop? The current docs say usb-ethernet only work after the above steps have been taken.

Cheers

**Comment by jedahan** [comment]
Oh, found some info here https://github.com/HclX/WyzeHacks/issues/146

And this generator https://codepen.io/ril3y/full/gXyzmO/

**Reply by granthopper** [reply]
Thanks for sharing this!
What firmware was your camera on that this worked for you?
I'm trying to get this to work on a Cam Pan v3.  I keep having trouble with being unable to SSH into the camera.  I suspect the issue is the newer firmware.  I am trying to find "recovery_wyzepan3_4.36.9.139.bin.zip"  as it appears that's the last version that works with wz_mini_hacks.
